ui/pages/bridge/bridge.util.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  BridgeFeatureFlagsKey,
+  BridgeFeatureFlags,
+} from '../../../app/scripts/controllers/bridge';
+import { BRIDGE_API_BASE_URL } from '../../../shared/constants/bridge';
+import { SWAPS_CLIENT_ID } from '../../../shared/constants/swaps';
+import fetchWithCache from '../../../shared/lib/fetch-with-cache';
+import { validateData } from '../../../shared/lib/swaps-utils';
+
+const CLIENT_ID_HEADER = { 'X-Client-Id': SWAPS_CLIENT_ID };
+const CACHE_REFRESH_TEN_MINUTES = 600000;
```

Comment:
Nit: you can use `10 * MINUTE` for clarity here

Reply:
good call!